Node/codeceptjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "codeceptjs",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "codeceptjs": "rimraf ./a11y-results && codeceptjs run --steps",
+        "codeceptjs:headless": "HEADLESS=true codeceptjs run --steps",
+        "codeceptjs:ui": "codecept-ui --app",
+        "codeceptjs:demo": "codeceptjs run --steps -c node_modules/@codeceptjs/examples",
+        "codeceptjs:demo:headless": "HEADLESS=true codeceptjs run --steps -c node_modules/@codeceptjs/examples",
+        "codeceptjs:demo:ui": "codecept-ui --app  -c node_modules/@codeceptjs/examples"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
```

Comment:
I'm not sure what our licenses are but I don't think its ISC we may need to have it `UNLICENSED` with our deque copyright